lib/src/constants.dart

```diff
@@ -0,0 +1,2 @@
+// TODO: Add link to upgrade guide.
+const String v3Deprecation = 'in 4.0.0.';
```

Comment:
The todo will be taken care as a part of this: https://github.com/Workiva/w_transport/issues/162